src/app/(drawerPages)/badges/[id]/page.tsx

```diff
@@ -45,61 +60,143 @@ export default function BadgeDetails({ params }: Props) {
     }
   }, [isDisconnected, router]);
 
+  useEffect(() => {
+    drawerStates.badgeActions.bottom = true;
+  }, [currentBadge, drawerStates.badgeActions]);
+
+  const ctaButton = useMemo(() => {
+    if (isOwned) return null;
+    if (!currentBadge) return null;
+    if (currentBadge.type === BadgeTypeEnum.IRL) {
+      return null;
+    } else if (currentBadge.type === BadgeTypeEnum.Online) {
+      return (
+        <Button variant="contained" href={currentBadge.ctaUrl}>
+          Mint
+        </Button>
+      );
+    }
+  }, [currentBadge, isOwned]);
+
+  const content = useMemo(() => {
+    if (currentBadge) {
+      const title = currentBadge?.name;
+      const [lat, lng] = currentBadge.latLng
```

Comment:
online badge wont' have lat_lng. It will be null

Reply:
fixed